sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AzureADGraphClient.java

```diff
@@ -184,16 +184,15 @@ private boolean isMatchingUserGroupKey(final JsonNode node) {
     /**
      * Determines if this is a valid {@link UserGroup} to build to a GrantedAuthority.
      * <p>
-     * If the {@link AADAuthenticationProperties.UserGroupProperties#getAllowedGroups()} or the {@link
-     * AADAuthenticationProperties#getActiveDirectoryGroups()} contains the {@link UserGroup#getDisplayName()} return
+     * If the {@link AADAuthenticationProperties.UserGroupProperties#getAllowedGroups()}
+     *  contains the {@link UserGroup#getDisplayName()} return
      * true.
      *
      * @param group - User Group to check if valid to grant an authority to.
-     * @return true if either of the allowed-groups or active-directory-groups contains the UserGroup display name
+     * @return true if either of allowed-groups contains the UserGroup display name
```

Comment:
'either of' is now not needed if only allowed-groups is being checked

Reply:
@hamishlawson yes, thanks for your suggestion.